### PR TITLE
#1582 request - updated pt_PT translation for 0.13

### PR DESCRIPTION
**Pull request by rmbb**
#1582 request - updated pt_PT translation

### CODE REVIEW

**Comment by erciccione** [comment]
@mrbeast87 Maybe you could review?

File: translations/monero-core_pt-pt.ts [hunk 1]
```diff
@@ -1079,235 +1079,242 @@ The following information will be deleted
...
         <location filename="../pages/settings/SettingsNode.qml" line="232"/>
         <source>Uses a third-party server to connect to the Monero network. Less secure, but easier on your computer.</source>
-        <translation type="unfinished"></translation>
+        <translation>Utiliza um servidor remoto para ligar à rede Monero. É um método menos seguro, mas consome menos armazenagem local.</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="277"/>
         <source>To find a remote node, type &apos;Monero remote node&apos; into your favorite search engine. Please ensure the node is run by a trusted third-party.</source>
-        <translation type="unfinished"></translation>
+        <translation>Para encontrar um node remoto, escreva &apos;node remoto de Monero&pos; no seu motor de busca favorito. Confirme sempre que o node é gerido por uma entidade confiável.</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: Miswritten entity `&pos;`, should be `&apos;`

File: translations/monero-core_pt-pt.ts [hunk 2]
```diff
@@ -1867,17 +1874,17 @@ The old wallet cache file will be renamed and can be restored later.
     <message>
         <location filename="../pages/Transfer.qml" line="191"/>
         <source>Automatic</source>
-        <translation type="unfinished"></translation>
+        <translation type>Automático</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: forgotten `type`. This should be `<translation>Automático</translation>`

File: translations/monero-core_pt-pt.ts [hunk 3]
```diff
@@ -2128,7 +2135,7 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
     <message>
         <location filename="../wizard/WizardCreateWalletFromDevice.qml" line="112"/>
         <source>Create a new wallet from hardware device</source>
-        <translation type="unfinished"></translation>
+        <translation type>Criar uma nova carteira a partir do dispositivo de hardware</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: as above, forgotten `type`

File: translations/monero-core_pt-pt.ts [hunk 4]
```diff
@@ -149,42 +149,42 @@
...
         <location filename="../pages/History.qml" line="237"/>
         <source>Date</source>
-        <translation type="unfinished">Data</translation>
+        <translation>Data</translation>
     </message>
     <message>
         <location filename="../pages/History.qml" line="278"/>
         <source>No history...</source>
-        <translation type="unfinished"></translation>
+        <translation>Sem histórico</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: Please add `...` at the end of the string

**Comment by rmbb** [comment]
Hi @erciccione,

Thanks for pointing out those errors.

Just fixed them in my repo.

I have no idea how to squash and commit, so if you can do it, go ahead!

Thanks and have a good weekend.

MB

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by mrbeast87** [comment]
Hey,

Cheers to @rmbb for his great work on this translation.

As requested by @erciccione, I reviewed the changes and suggest the following improvements:

> line 986: there is an unnecessary parenthesis at the end of line.
> line 1158: "Garante maior segurança e também mais espaço disponivel de armazenagem." change to "Garante maior segurança mas necessita de mais espaço de armazenamento."
> line 1168: change all instances of "armazenagem" to "armazenamento", which is the correct term in pt_pt.
> line 1347: fix typo: emfraqiecer > enfraquecer
> line 1418: perca > perda 

Maybe I could have done this changes directly in the file?

Sorry but I'm still a noob using GitHub :P

**Comment by erciccione** [comment]
@rmbb please include @mrbeast87's review. Next release will be tagged soon and we need all translations to be ready.

@mrbeast87 If for some reason your changes won't be added, you can open a Pull Request after this one will get merged

**Comment by rmbb** [comment]
@mrbeast87  - file has been updated on my repo according to your suggestions, thanks a lot! 
@erciccione can you squash and commit the new PR of the new file?

Thanks.

**Pull request merged**